pkg/domain/infra/abi/system.go

```diff
@@ -123,7 +123,7 @@ func (ic *ContainerEngine) SetupRootless(_ context.Context, cmd *cobra.Command)
 		}
 	}
 	if err != nil {
-		logrus.Error(err)
+		logrus.Error(errors.Wrapf(err, "invalid internal status, try resetting the pause process with %q", "system migrate"))
```

Comment:
Just curious, why didn't you do:
```suggestion
		logrus.Error(errors.Wrapf(err, "invalid internal status, try resetting the pause process with system migrate"))
```

Reply:
I assume system migrate should be in quotes.  
`logrus.Error(errors.Wrapf(err, "invalid internal status, try resetting the pause process with \"system migrate\""))`
is another method.

Reply:
Would this be better with 
logrus.Error(errors.Wrapf(err, "invalid internal status, try resetting the pause process with %q", "os.Args[0] system migrate"))

Reply:
I've used that to make clearer what part is quoted

Reply:
changed to `logrus.Error(errors.Wrapf(err, "invalid internal status, try resetting the pause process with %q", os.Args[0]+" system migrate"))`